src/frontend/src/flows/register/finish.ts

```diff
@@ -18,21 +24,28 @@ const pageContent = (userNumber: bigint) => html`
       ${userNumber}
     </div>
     <div class="l-stack">
-      <button id="displayUserContinue" class="c-button">Continue</button>
+      <button
+        @click=${() => onContinue()}
+        id="displayUserContinue"
+        class="c-button"
+      >
+        Continue
+      </button>
     </div>
   </div>
 `;
 
-export const displayUserNumber = async (userNumber: bigint): Promise<void> => {
-  const container = document.getElementById("pageContent") as HTMLElement;
-  render(pageContent(userNumber), container);
-  return init();
+export const displayUserNumberPage = (
+  props: Parameters<typeof displayUserNumberTemplate>[0],
```

Comment:
actually never seen - I think - `[0]` used to express a type, does that work? nice to know

Reply:
yep, in this case that takes the type of the first param!

Reply:
cool